src/guess_density.jl

```diff
@@ -32,6 +32,63 @@ guess_density(basis::PlaneWaveBasis) = guess_density(basis, basis.model.atoms)
     from_fourier(basis, ρ / sqrt(model.unit_cell_volume))
 end
 
+
+@doc raw"""
+    guess_spin_density(basis, magnetic_moments)
+
+Magnetic moments should be specified in units of ``μ_B / 2``
+"""
+function guess_spin_density(basis::PlaneWaveBasis, magnetic_moments=[])
+    guess_spin_density(basis, basis.model.atoms, magnetic_moments)
+end
+@timing function guess_spin_density(basis::PlaneWaveBasis{T}, atoms, magnetic_moments) where {T}
+    # TODO Code duplication with guess_density
```

Comment:
One way to merge is to have a `superposition_atomic_densities` with a `weight` parameter.